ark/utils/load_utils.py

```diff
@@ -65,6 +66,9 @@ def load_imgs_from_mibitiff(data_dir, mibitiff_files=None, channels=None, delimi
         channel_tuples = tiff.read(mibitiff_files[0]).channels
         channels = [channel_tuple[1] for channel_tuple in channel_tuples]
 
+    if len(channels) == 0:
```

Comment:
We've already checked that mibitiff_files has at least one valid entry, I don't think this error makes sense?

Reply:
Even if `mibitiff_files` has at least one valid entry, it will fail if the user passes `[]` for `channels`. This will happen in `Segment_Image_Data` if both `nucs` and `mems` are set to `None`.

Reply:
Got it, in that case we should change the error message